ansible/playbooks/roles/compatibility_layer/defaults/main.yml

```diff
@@ -18,17 +18,26 @@ gentoo_prefix_path: /cvmfs/{{ cvmfs_repository }}/{{ eessi_version }}/compat/{{
 # How to build the prefix
 prefix_required_space: 15 GB
 prefix_snapshot_url: https://eessi-gentoo-snapshot.s3-eu-west-1.amazonaws.com
-prefix_snapshot_version: 20210607
+prefix_snapshot_version: 20211120
 prefix_user_defined_trusted_dirs:
   - "/cvmfs/{{ cvmfs_repository }}/host_injections/{{ eessi_version }}/compat/{{ eessi_host_os }}/{{ eessi_host_arch }}/lib"
-prefix_singularity_command: "singularity run -B {{ gentoo_prefix_path }}:{{ gentoo_prefix_path }}"
+prefix_mask_packages: |
+  # Mask glibc >= 2.34, as it's causing issues with the bootstrap, and it's not compatible with CUDA 11.
+  >=sys-libs/glibc-2.34
```

Comment:
@bedroge I'm wondering whether we should use `==` to mask these problematic versions for both `glibc` and `libgcrypt` (see also the proposed changes in https://github.com/bedroge/compatibility-layer/pull/2)

Since we also fix the Portage snapshot we're using, that should work just as well to avoid these problematic versions.

The benefit is that we won't accidentally keep masking anything newer than glibc 2.34 (likewise for libgcrypt), even when working on a new compat layer.
It would be very easy to overlook this masking stuff for a later version of the compat layer when using `>=`, I think...

Reply:
Yes, sounds like a good idea, I've changed it for both packages.